packages/@o3r/schematics/src/rule-factories/ng-add/ng-add.helpers.ts

```diff
@@ -121,7 +120,10 @@ export function ngAddPackages(packages: string[], options?: Omit<NgAddPackageOpt
         hideOutput: false,
         quiet: false
       } as any));
-      await lastValueFrom(context.engine.executePostTasks());
+
+      await new Promise<void>((resolve, reject) =>
```

Comment:
why is that needed?
by the look of it, the only difference is that we hide the exception on error